tools/src/tester/SchemaValidator.ts

```diff
@@ -25,6 +31,10 @@ export default class SchemaValidator {
   validate (schema: OpenAPIV3.SchemaObject, data: any): Evaluation {
     const validate = this.ajv.compile(schema)
     const valid = validate(data)
+    if (! valid) {
+      this.logger.info(`# ${JSON.stringify(schema, null, 2)}`)
+      this.logger.info(`* ${JSON.stringify(data, null, 2)}`)
```

Comment:
Nit: this is used often enough to DRY it with `to_json` function in `helpers.ts` and to keep the JSON format consistent throughout the framework.

Reply:
Good idea, https://github.com/opensearch-project/opensearch-api-specification/pull/342/commits/b8484ac83bcb7ccdc3ccf0987e158893917adae2.